app/service/form_list_service.rb

```diff
@@ -16,6 +15,12 @@ def initialize(forms:, current_user:, search_form: nil)
     @forms = forms
     @current_user = current_user
     @search_form = search_form
+    if current_user.super_admin?
+      @list_of_creator_id = forms.map(&:creator_id).uniq
+      @list_of_creators = User.where(id: @list_of_creator_id)
+                               .select(:id, :name)
+                               .map { |user| { id: user.id, name: user.name } }
+    end
```

Comment:
I think this creates queries like this when there is a form which doesn't have a creator set (which I think might still be possible?)

```sql
SELECT "users"."id", "users"."name" FROM "users" WHERE ("users"."id" IN ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11, $12, $13, $14, $15, $16, $17, $18, $19, $20, $21, $22) OR "users"."id" IS NULL)
```

I don't think the `OR "users"."id" IS NULL)` matters because we should. never have a user without an id?

if it does we could add `.compact` to line 19.

Should list_of creators be a __list__ of creators? Would a hash be better? e.g. 

```ruby
User.where(id: @list_of_creator_id).select(:id, :name).pluck(:id, :name).to_h
```
This would simplify find_creator_name a bit.

This is a tiny optimisation though and might be over doing it :)